src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/Semantic/DefaultRazorSemanticTokenInfoService.cs

```diff
@@ -22,28 +26,27 @@ public override SemanticTokens GetSemanticTokens(RazorCodeDocument codeDocument,
             {
                 throw new ArgumentNullException(nameof(codeDocument));
             }
-            var syntaxTree = codeDocument.GetSyntaxTree();
 
-            var syntaxNodes = VisitAllNodes(syntaxTree);
+            var syntaxNodes = VisitAllNodes(codeDocument);
```

Comment:
This no longer returns SyntaxNodes. We should call it syntaxRanges or syntaxRangeResults or similar. Ditto for the actual type.